pages/about/product_updates.md

```diff
@@ -0,0 +1,33 @@
+---
+title: .Gov product updates
+permalink: /about/product/
+layout: layouts/info-page
+sidenav: true
+excerpt: Learn about .gov product updates
+tags: about
+eleventyNavigation:
+  key: about
+  order: 3
+  title: .Gov product updates
+---  
+
+We’ll share notes about product releases and new features here.
+
+## November 17, 2023
+
+Our new domain management system—the .gov registrar—is now available. The organization, contacts, and DNS information for all .gov domains have been transferred to the new registrar. 
+
+If you have questions or concerns about the information you see for your domain, email us at help@get.gov.
+
+### Changes from the previous registrar
+- We no longer have the admin, technical, and billing roles. In the new registrar, these contacts are all considered “domain managers,” and you aren’t limited to three domain managers like in the old system.
+- We’re now using Login.gov to manage access to the registrar. Login.gov is a simple and secure process for signing in to many government services with one account.
+
+### Some domain information won’t immediately be available in the registrar
+- Domain expiration date: Good news! Expiration dates for .gov domains in good standing have been extended for one year.
```

Comment:
We haven't actually done this yet but we will! It can go in the "aren't enabled yet" section